sdks/java/nexmark/src/main/java/org/apache/beam/sdk/nexmark/model/Auction.java

```diff
@@ -80,14 +80,14 @@ public Auction decode(
 
   /** Extra auction properties. */
   @JsonProperty
-  private final String itemName;
+  public final String itemName;
```

Comment:
I mentioned `@AutoValue` before and I think you can do this, at least for Jackson:

```java
@AutoValue
@AutoValue.CopyAnnotations
public abstract class Auction {
  @JsonProperty
  public abstract String getItemName();
}
```

I don't know that it is possible to use `@AutoValue` with avro schema generation, though.